stylesheet/components/cart.css

```diff
@@ -0,0 +1,53 @@
+.cart-container{
+     margin:  0 auto;
+}
+
+table{
+     width: 100%;
+     border-collapse: collapse;
+}
+
+.product-detail{
+     display: flex;
+     flex-wrap: wrap;
```

Comment:
I think you should create this utility class. That will help full for you in most of component later .

Reply:
okay i ll do it